src/tools/config/getConfig.ts

```diff
@@ -12,9 +16,29 @@ export const getConfig = (name: string): BackendConfigType => {
   // to the new format.
   const config = readConfig(name);
   if (config.schemaVersion === undefined) {
-    Consola.log('Migrating config file...', config);
+    Consola.log('Migrating config file...', config.name);
     return backendMigrateConfig(config, name);
   }
 
-  return config;
+  let backendConfig = config as BackendConfigType;
+
+  if (backendConfig.widgets.some((widget) => !uuidRegex.test(widget.id))) {
+    backendConfig = {
+      ...backendConfig,
+      widgets: backendConfig.widgets.map((widget) => ({
+        ...widget,
+        id: uuidv4(),
```

Comment:
I should check the id here again. Because if only one does not match a uuid the config will be screwed